insurance_cloudevents/README.md

```diff
@@ -0,0 +1,90 @@
+# INSURANCE EXAMPLE
+
+- This example only supports predict dataset sources as **CloudEvents**. 
+
+- The notebooks in this example can be run inside or outside Dkube.
+
+## Example Flow
+- Create DKube resources. This includes Dataset and Model repo resources.
+- Train a model for Insurance example using SkLearn and deploy the model for inference.
+- Create a Modelmonitor. 
+  - There is a mandatory requirement to deploy a model for this example
+- Generate data for analysis by Modelmonitor
+  - Predict data: Inference inputs/outputs
+  - Label data:  Dataset with Groundtruth values for the inferences made above
+  **In production, Label data would be generated by experts in the domain manually.**
+- Cleanup resources after the example is complete
+
+## Section 1: Create Dkube Resources
+
+### Launch IDE (Inside Dkube)
+
+#### Note: Follow the instructions if you are running Notebook IDE inside DKube.
+
+1. Add Code. Create Code Repo in Dkube with the following information
+  - Name: insurance
+  - Source: Git
+  - URL: https://github.com/oneconvergence/dkube-examples.git
+  - Branch : **monitoring**
+2. Create an IDE (JupyterLab)
+   - Use sklearn framework  
+3. Click Submit.
+4. Open Jupyterlab and from **workspace/insurance/insurance_cloudevents** open resources.ipynb and fill the following details in the first cell. 
+     - **DKUBEUSERNAME** = {your dkube username}
+     - **MODELMONITOR_NAME** = {your model monitor name}
+     - **S3_KEY** = {MINIO access key}
+     - **S3_SECRET_KEY** = {MINIO access secret key}
+     - **S3_ENDPOINT** = {MINIO Endpoint}
+     - The following will be derived from the environment automatically if the notebook is running inside same Dkube IDE. Otherwise, please fill in 
+       - **TOKEN** = {your dkube authentication token}
+       - **DKUBE_URL** = {your dkube url}
+5. Run all the cells. This will create all the dkube resources required for this example automatically.
+
+### Launch IDE (Outside Dkube)
+
+#### Note: Follow the instructions if you are running Notebook IDE outside DKube, for example in VSCode. This is the case with minimal DKube. 
+
+1. Download [resources.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/insurance/resources.ipynb)
+2. Open the notebook and fill the details in the first cell.
+   - **DKUBEUSERNAME** = {your dkube username}
+   - **MODELMONITOR_NAME** = {your model monitor name}
+   - **TOKEN** = {your dkube authentication token}
+   - **DKUBE_URL** = {your dkube url}
+   - **S3_KEY** = {MINIO access key}
+     - **S3_SECRET_KEY** = {MINIO access secret key}
+     - **S3_ENDPOINT** = {MINIO Endpoint}
+3. Run all the cells.
+
+## Section 2: Insurance Model Training
```

Comment:
Can we make this optional?

Reply:
Without training, the deployment will not be possible.